lib/my_app.dart

```diff
@@ -15,7 +28,18 @@ class MyApp extends StatelessWidget {
       ),
       initialRoute: MovieListHome.routeName,
       routes: {
-        MovieListHome.routeName: (context) => const MovieListHome(),
+        MovieListHome.routeName: (context) => MovieListHome(
+              blocMovieList: BlocMovieList(
```

Comment:
Why you call it BlocMovieList, the bloc class never has a reference of a ui structure

Reply:
fixed